frontend/src/components/Cards/CardStore.js

```diff
@@ -1,106 +1,110 @@
-import React from 'react';
+import React from "react";
 import Card from "react-bootstrap/Card";
-import './CardStore.css'
-import Button from 'react-bootstrap/Button';
+import "./CardStore.css";
+import Button from "react-bootstrap/Button";
 import GameInstance from "../../services/gameService";
 
 class CardStore extends React.Component {
-    constructor(props, context) {
-        super(props, context);
-
-        this.handleChange = this.handleChange.bind(this);
-        this.clearSelected = this.clearSelected.bind(this);
-
-        this.state = {
-            username: props.currentUser,
-            gameRoom: props.currentRoom,
-            value: [],
-            selectedCard: [],
-            cardName: 'Card Name',
-            cardCost: 3,
-            cardType: 'keep',
-            cardEffect: 'lose two hearts, gain 4 energy',
-            cardFootnote: 'footnote description',
-
-        };
-    }
+  constructor(props, context) {
+    super(props, context);
+
+    this.handleChange = this.handleChange.bind(this);
+    this.clearSelected = this.clearSelected.bind(this);
+
```

Comment:
This is now no longer about the backend stuff and should likely have been separated into a separate pull request.

Reply:
yep, inherited. Defn would not be in final PR ... unless its in master.